rpc/src/rpc.rs

```diff
@@ -1095,7 +1097,16 @@ impl JsonRpcRequestProcessor {
             {
                 self.check_blockstore_writes_complete(slot)?;
                 let result = self.blockstore.get_rooted_block(slot, true);
-                self.check_blockstore_root(&result, slot)?;
+
+                let do_check_blockstore_root: bool = if let Some(rpc_bigtable_config) = &self.config.rpc_bigtable_config {
+                    !rpc_bigtable_config.skip_blockstore_root_check
+                } else {
+                    false
```

Comment:
It should be `true` by default. If the user provides `--rpc-bigtable-skip-blockstore`, we skip this check.
```suggestion
                    true
```

Reply:
@dragvs